### PR TITLE
[T4A4][W11-B3] Fan Weiguang

### PR DESCRIPTION
**Pull request by fanwgwg**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,8 +6,9 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable{
 
+	public static final String ADDRESS_PREFIX = "Address: ";
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Address.java]: Please fix the indentation and change the tab to spaces.

Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Address.java]: Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -164,5 +165,16 @@ private static String getIndexedListForViewing(List<String> listItems) {
...
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    private String getPrintableString(Printable... printables) {
+    	final StringBuilder formatted = new StringBuilder();
+    	for(Printable i : printables) {
+    		formatted.append(i.getPrintableString()).append("\n");
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/ui/TextUi.java]: Be aware that there are a few ways to concatenate strings in Java, such as `String.join`, etc.

File: src/seedu/addressbook/data/person/Phone.java [hunk 3]
```diff
@@ -56,4 +57,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString() {
+		return PHONE_PREFIX + this.value;
+	}
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Phone.java]: Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments.

**Comment by fanwgwg** [comment]
Comments reviewed. I will fix and formatter. Thank you very much!

**Pull request closed without merging**